### ISSUE TITLE
Easier comparison of HANDLE to INVALID_HANDLE_VALUE

### ISSUE THREAD

**Issue by eminence** [microsoft/windows-rs#650]
Hello!

`windows::win32::system_services::HANDLE` wraps an isize, but `windows::win32::system_services::INVALID_HANDLE_VALUE` is an i32, making a comparison require a field access and a cast.

```rust
let h_out = unsafe { GetStdHandle(STD_HANDLE_TYPE::STD_OUTPUT_HANDLE) };
if h_out.0 == INVALID_HANDLE_VALUE as isize {
    return;
}
```

Any chance these could be directly comparable?

Thanks!

**Comment by Robo210** [comment]
INVALID_HANDLE_VALUE is supposed to be pointer sized to begin with:

```C
#define INVALID_HANDLE_VALUE ((HANDLE)(LONG_PTR)-1)
```

**Comment by kennykerr** [comment]
This has been fixed as of #642. Here's an example:

https://github.com/microsoft/windows-rs/blob/81491f85f196f7cc6f2030bf4e50de3456283357/examples/overlapped/src/main.rs#L11-L23
